xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_bg.properties

```diff
@@ -3644,7 +3653,7 @@ core.menu.userDashboard=My dashboard
 ### Missing: extensions.uninstall.error.prepareFailure=Failed to prepare uninstalling extension with id {0} and version {1}:
 ### Missing: extensions.uninstall.error.notInstalled=This extension is not installed.
 ### Missing: extensions.uninstall.cleanPages.label=Delete unused wiki pages?
-### Missing: extensions.uninstall.cleanPages.hint=The following wiki pages are not needed any more so it should be safe to delete them. Unselect the ones that you wish to keep. The wiki pages that have modifications are left unselected so that you don't loose your changes. Select them if those changes are not important.
+extensions.uninstall.cleanPages.hint=The following wiki pages are not needed any more so it should be safe to delete them. Unselect the ones that you wish to keep. The wiki pages that have modifications are left unselected so that you don't loose your changes. Select them if those changes are not important.
```

Comment:
This is wrong apparently all languages where the key was not translated, apparently automatically translated it using the old value...